lib/state/app_state.dart

```diff
@@ -8,7 +8,7 @@ part 'app_state.g.dart';
 class AppState with _$AppState {
   factory AppState({
     /// TODO: this is for testing purposes and will be changed once the state is finalized
-    @Default('') @JsonKey(ignore: true) String name,
+    @JsonKey(ignore: true) String? pokemons,
```

Comment:
Now my next question would be, is there any reason why the default annotation is removed and made this state property nullable? 🤔

Reply:
there is no particular reason I just thought that it may result to null but I think I should just revert it

Reply:
have it reverted then, that's the use of the `@Default` annotation to supply with a default value, meaning in this case it won't get null unless you provided a null default value 🙂